src/commands/secret/mod.rs

```diff
@@ -35,25 +36,50 @@ fn validate_target(target: &Target) -> Result<(), failure::Error> {
 fn secret_errors(error_code: u16) -> &'static str {
     match error_code {
         7003 | 7000 => {
-            "Your wrangler.toml is likely missing the field \"account_id\", which is required to write to Workers KV."
+            "Your wrangler.toml is likely missing the field \"account_id\", which is required to create a secret."
         }
         10053 => "There is already another binding with a different type by this name. Check your wrangler.toml or your Cloudflare dashboard for conflicting bindings",
-        10054 => "Your secret is too large, bindings must be 1kB or less",
+        10054 => "Your secret is too large, it must be 1kB or less",
         10055 => "You have exceeded the limit of 32 text bindings for this worker. Run `wrangler secret list` or go to your Cloudflare dashboard to clean up unused text/secret variables",
         _ => "",
     }
 }
 
+// upload_draft_worker will attempt to upload a "draft" version of a worker script if it does not
+// already exist in the API (API error code 10007 is returned). The function returns None if this draft
+// script was uploaded, or else returns Some (with a Result type so we can return a potential script upload error
+// up the call chain).
+pub fn upload_draft_worker(
+    e: &ApiFailure,
+    user: &GlobalUser,
+    target: &Target,
+) -> Option<Result<(), failure::Error>> {
+    match e {
+        ApiFailure::Error(_, api_errors) => {
+            for error in &api_errors.errors {
+                if error.code == 10007 {
+                    message::working("This worker doesn't exist in the API yet. Uploading worker script so we can create new secret.");
```

Comment:
nit: (warning: not `cargo check`-ed)
```suggestion
                    message::working(&fmt("Worker {} doesn't exist yet. Creating a draft to attach secret to.", target.name));
```